src/tree-view/treeViewComponent.ts

```diff
@@ -0,0 +1,81 @@
+const $ = (<any> window).$;
+const Promise = (<any> window).Promise;
```

Comment:
No need to use these constants. `$` is registered via webpack as public package, and Promise is ES6 thing which comes with typescript.